hardware/src/base-station-sketch2/CmdMessenger.cpp

```diff
@@ -0,0 +1,680 @@
+/*
+  CmdMessenger - library that provides command based messaging
```

Comment:
This and the `.h` file need to be included in the same directory as the `.ino`. Not sure if there is a work around for that.